docs/compass/tenant-loading.md

```diff
@@ -0,0 +1,69 @@
+# Initial tenant loading
+
+## Overview
+This document describes how Compass handles initial tenant loading.
+
+## Job
+
+In Compass the user can define his own tenants. There is a special `compass-tenant-loader` job provided for that operation. User has to define a JSON file which looks like that
+```json
+[
+  {
+    "name": "tenant-name-1",
+    "id": "tenant-id-1"
+  },
+  {
+    "name": "tenant-name-2",
+    "id": "tenant-id-2"
+  }
+]
+```
+The JSON file has to be added to `compass-tenant-config` ConfigMap defined in `chart/compass/charts/director/templates/configmap-tenant-config.yaml` where you have to put your file into the data object.
+
+Example ConfigMap:
+```helmyaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: compass-tenant-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Chart.Name }}
+    release: {{ .Release.Name }}
+data:
+    {{- tpl ((.Files.Glob "tenants.json").AsConfig) . | nindent 2 }}
```

Comment:
does it mean that the user has to modify the chart manually?

Reply:
addressed that issue